src/components/Admin.tsx

```diff
@@ -26,41 +30,72 @@ const Admin = () => {
             <li>
               <Button
                 appearance="transparent"
-                onClick={() =>
-                  seturl(
-                    _spPageContextInfo.webAbsoluteUrl +
-                      "/Lists/Disclaimers/AllItems.aspx"
-                  )
-                }
+                onClick={() => seturl(globalList + "Disclaimers/AllItems.aspx")}
               >
                 Disclaimer Statements (Global)
               </Button>
             </li>
-            <li>Department of Defense Activity Address Directory (DoDAAD)</li>
+            <li>
+              <Button
+                appearance="transparent"
+                onClick={() => seturl(globalList + "DODAAC/AllItems.aspx")}
+              >
+                DODAACs
+              </Button>
+            </li>
           </ul>
         </>
       )}
       {program && programPerms.data && (
         <>
           <Title3>{program} Program Office Items</Title3>
           <ul>
-            <li>Contractors</li>
-            <li>Contracts</li>
             <li>
               <Button
                 appearance="transparent"
                 onClick={() =>
-                  seturl(
-                    _spPageContextInfo.webAbsoluteUrl +
-                      "/" +
-                      program +
-                      "/Lists/Disclaimers/AllItems.aspx"
-                  )
+                  seturl(programList + "Contractors/AllItems.aspx")
+                }
+              >
+                Contractors
+              </Button>
+            </li>
+            <li>
+              <Button
+                appearance="transparent"
+                onClick={() => seturl(programList + "Contracts/AllItems.aspx")}
+              >
+                Contracts
+              </Button>
+            </li>
+            <li>
+              <Button
+                appearance="transparent"
+                onClick={() =>
+                  seturl(programList + "ContractorPOCs/AllItems.aspx")
+                }
+              >
+                Contractor POCs
```

Comment:
The iframe might need to be put inside a popup -- in order to allow it to fill the screen -- so that editing can be a bit better, as it gets tight being in just the bottom portion.
![alt text](image.png)

Reply:
I'd have to see it to know for sure :) -- Also we have to make sure we test it being viewed on a "normal" width screen.